Makefile

```diff
@@ -46,30 +46,28 @@ endif
 # "apiserver" instead of "bin/apiserver".
 #########################################################################
 build: .init .generate_files \
-       $(BINDIR)/controller $(BINDIR)/registry $(BINDIR)/k8s-broker \
-       $(BINDIR)/service-catalog $(BINDIR)/user-broker \
-       $(BINDIR)/apiserver
+       $(BINDIR)/controller $(BINDIR)/apiserver \
+       $(BINDIR)/registry $(BINDIR)/k8s-broker $(BINDIR)/user-broker
```

Comment:
Along with `.init .generate_files`, these are the pre-requisite target for the `build` target. `$(BINDIR)/service-catalog` went away because it seemed to be a duplicate of the controller and as for the rest, I just grouped targets related to service catalog components on one line and contrib stuffs on another.